unittest/buildtool/container_commands_test.py

```diff
@@ -61,6 +62,7 @@ def test_tag_containers_dry_run(self):
             os.path.dirname(__file__), "standard_test_bom.yml"
         )
         options.dry_run = True
+        options.tag_java11 = "gate"
```

Comment:
We can set this `True`.

The `standard_test_bom.yml` file has a subset of services so the rest of the call assertions should be ok.